components/eam/src/chemistry/aerosol/wetdep.F90

```diff
@@ -12,6 +12,7 @@ module wetdep
 use phys_control, only: cam_physpkg_is
 use cam_logfile,  only: iulog
 use cam_abortutils, only: endrun
+use spmd_utils,   only: masterproc
 
 implicit none
 save
```

Comment:
@singhbalwinder I see there is a global `save` .  Would it be possible to determine which local variables need to be saved and mark them individually?  It would be useful to know for the C++ version.

Reply:
Thanks, @overfelt! This is a very good comment. We should be careful about where the variable values are coming from. Fortunately, in this case, we are not using any `save`ed variables (although the variables are declared at the module level). We will add checking and marking such variables as a requirement the next time.